### PR TITLE
✏ Fix typos and add rewording in docs

### PR DESCRIPTION
**Pull request by nukopy**
Fix typos in docs.

## target

- "Tutorial - User Guide"
  - "Request Body"
  - "Query Parameters and String Validations"
  - "Path Parameters and Numeric Validations"

### CODE REVIEW

File: docs/en/docs/tutorial/path-params-numeric-validations.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ And you can also declare numeric validations:
 * `le`: `l`ess than or `e`qual
 
 !!! info
-    `Query`, `Path` and others you will see later subclasses of a common `Param` class (that you don't need to use).
+    `Query`, `Path`, and others you will see later are subclasses of a common `Param` class (that you don't need to use).
```
Comment (ycd) [inline docs/en/docs/tutorial/path-params-numeric-validations.md:109]: Why do we need a comma after "Path" when we already have "and".

Comment (nukopy) [reply docs/en/docs/tutorial/path-params-numeric-validations.md:109]: This is a grammatical issue.
To be honest, I don't know how to use commas in this situation.

I don't think the comma in this case should be omitted in either American English or British English. Therefore, I add a comma.

- A, B and C
- **BETTER: A, B, and C**

How is it practically used?

Comment (ycd) [reply docs/en/docs/tutorial/path-params-numeric-validations.md:109]: Do not get me wrong your PR is perfectly fine by me, I was just curious as a non-native speaker.

I found [this](https://en.wikipedia.org/wiki/Serial_comma). Seems like it is very common for native speakers to put a comma before the last item in a list. They are calling it the Oxford Comma.

**Review by ycd — APPROVED** [review]

**Review by ycd — APPROVED** [review]

File: docs/en/docs/tutorial/path-params-numeric-validations.md (hunk 1 above)
Comment (ycd) [reply docs/en/docs/tutorial/path-params-numeric-validations.md:109]: Feel free to mark the conversation as resolved.

Comment (nukopy) [reply docs/en/docs/tutorial/path-params-numeric-validations.md:109]: Thanks for your comments, @ycd.
I'm a non-native speaker, too.
I learned a lot from you.

**Comment by tiangolo** [comment]
Awesome, thank you @nukopy ! :rocket: :tada: 

And thanks @ycd for the help here! :bow: :cake:

**Pull request merged**